website/docs/core-concepts/media.md

```diff
@@ -4,4 +4,21 @@ sidebar_position: 7
 
 # Media
 
-## Coming Soon!
+Media to be used across the space can be found here. Media uploaded from any other place across Dega is also listed here.
+Admin of an organization can limit the number of media for a space, which can be changed from the Space Requests menu by making a request to change the number of media for a space.
```

Comment:
Change to: 

The administrator of an organization can limit the number of media files allowed within a space. This limit can be configured from the `Space Requests` menu by creating a request to change the media file limit for a space.

Reply:
updated